addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1418,6 +1418,7 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
     _renderColorPalette: function () {
         const options = {
             selectedColor: this._value,
+            $editable: this.$target.closest('.o_editable'),
```

Comment:
@younn-o I merged this a little bit too fast yesterday, this is in fact not good enough
- `this.$target.closest('.o_editable')` will only return the editable container where the `$target` is, not all editable containers of the page, so you might miss some custom colors.
- `this.$target` should be avoided in `UserValueWidget`, it was added as a workaround for a specific usecase but it kinda breaks the logic of having widgets returning values, independantly of the DOM. I'd like to remove that at some point.

Could you re-improve this fix in 14.3? Probably by keeping the `request_editable` event and restoring its listener at `SnippetMenu` level? (since the upper level that handled it was removed by the editor team)

Reply:
Sorry about that! I pushed a fix for the fix here : https://github.com/odoo/odoo/pull/72978